pkg/transforms/http_test.go

```diff
@@ -103,10 +103,10 @@ func TestHTTPPostNoParameterPassed(t *testing.T) {
 func TestHTTPPostInvalidParameter(t *testing.T) {
 
 	sender := NewHTTPSender("", "", false)
-	data := 25
+	data := make(chan int)
```

Comment:
Please comment why using a channel.

Reply:
Channels are not marshalable to JSON and are the standard way to generate that error.